packages/firestore/package.json

```diff
@@ -30,7 +30,7 @@
     "dist"
   ],
   "dependencies": {
-    "@firebase/firestore-types": "1.5.0",
+    "@firebase/firestore-types": "file:../firestore-types/",
```

Comment:
No need to manually change package version. Lerna will make the new types available to all packages in the workspace in our dev environment.

Package versions are automatically updated by the release script during a release.

Reply:
I agree in principle, but in practice, `yarn test:browser` is failing from packages/firestore. 😄 It seems like with our current setup we have a period of brokenness between when we check in a change to firestore-types and when we do the next release.  Any suggestions?